src/settings/settingsMenuItems.component.tsx

```diff
@@ -25,7 +26,7 @@ const SettingsMenuItems = () => {
       }),
     []
   );
-
+  const { data: colourMaps } = useColourMaps();
```

Comment:
It's a minor thing, but pretty sure this could stay in the `ColourMapMenuItem` component. Maybe it's slightly more efficient it being here though as it saves having 2 react-query observers rather than 1